signalingFilePaths.cmake

```diff
@@ -7,11 +7,13 @@
 
 # Signaling library source files.
 set( SIGNALING_SOURCES
-     "${CMAKE_CURRENT_LIST_DIR}/source/signaling_api.c" )
+     "${CMAKE_CURRENT_LIST_DIR}/source/signaling_api.c" 
+     "${CMAKE_CURRENT_LIST_DIR}/source/dependency/coreJSON/source/core_json.c" )

```

Comment:
Can this move out separately?
Users who's using signaling lib might have his own core_json files so we shouldn't limit that posibility here.

Reply:
Sure will do it.